graph/pull_v2.go

```diff
@@ -502,7 +502,7 @@ func fixManifestLayers(m *schema1.Manifest) error {
 
 	if images[len(images)-1].Parent != "" && !allowBaseParentImage {
 		// Windows base layer can point to a base layer parent that is not in manifest.
-		return errors.New("Invalid parent ID in the base layer of the image.")
+		return errors.New("invalid parent ID in the base layer of the image.")
```

Comment:
missed one `graph/pull_v2.go:505:21: error strings should not end with punctuation`